go.mod

```diff
@@ -50,3 +54,15 @@ replace (
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/k8sclusterreceiver v0.0.0-00010101000000-000000000000 => github.com/open-telemetry/opentelemetry-collector-contrib/receiver/k8sclusterreceiver v0.15.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/redisreceiver v0.0.0-00010101000000-000000000000 => github.com/open-telemetry/opentelemetry-collector-contrib/receiver/redisreceiver v0.15.0
 )
+
+// each of these is required for the smartagentreceiver
+replace (
+	code.cloudfoundry.org/go-loggregator => github.com/signalfx/go-loggregator v1.0.1-0.20200205155641-5ba5ca92118d
+	github.com/dancannon/gorethink => gopkg.in/gorethink/gorethink.v4 v4.0.0
+	github.com/influxdata/telegraf => github.com/signalfx/telegraf v0.10.2-0.20201211214327-200738592ced
+	github.com/prometheus/prometheus => github.com/prometheus/prometheus v1.8.2-0.20201105135750-00f16d1ac3a4
+	github.com/signalfx/signalfx-agent => github.com/signalfx/signalfx-agent v1.0.1-0.20210104180832-f3147854c36c
```

Comment:
Can you please clarify version number used? Which SmartAgent version are we importing? The latest seems to be v5.7.1 but we use v1.0.1? I am probably misunderstanding something.

Reply:
The Smart Agent didn't follow the expected practice of a major version suffix for go modules (https://blog.golang.org/v2-go-modules) so I believe now only pseudo-versions are supported.  Without them, the latest version available is the last one before adopting go modules.  These versions are correct for each modules within the signalfx-agent project per `go get`.

Reply:
To clarify, the version is this commit: https://github.com/signalfx/signalfx-agent/commit/f3147854c36c45a7d73e52f4b94f57a83381df19.  It's required for the tests to pass.